special-interest-groups/sig-k8s/README.md

```diff
@@ -0,0 +1,27 @@
+# K8S Special Interest Group(K8S SIG)
+
+K8S SIG covers the TiDB Products on Kubernetes and Docker. Our work includes
+* Supporting new features or improving performance for TiDB Products on Kubernetes and Docker.
+* Supporting TiDB ecosystem tools intergrated natively with TiDB on Kubernetes
+
+## Roles and Organization Management
+
+See [SIG K8S Roles and Organization Management](./roles-and-organization-management.md)
+
+## Members
+
+See [SIG K8S Member List](./member-list.md)
+
+## Meetings
+
+* TBD
+
+## Contact
+
+* Slack: channel #sig-tidb-operator in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
```

Comment:
I don't think tidb-operator is proper for this sig, maybe tidb-on-cloud?

Reply:
I agreed. As #sig-tidb-cloud is an existed channel, we should rename it as #sig-tidb-on-cloud

Reply:
There is no such channel yet.